core/src/test/java/com/linecorp/armeria/server/HttpServerTest.java

```diff
@@ -894,6 +913,33 @@ public void testTrailers() throws Exception {
         assertThat(res.trailingHeaders().get(AsciiString.of("foo")), is("bar"));
     }
 
+    @Test(timeout = 10000)
+    public void testExactPathCached() throws Exception {
+        org.assertj.core.api.Assertions.assertThat(client().get("/cached-exact-path")
```

Comment:
Would you mind just replacing hamcrest with assertj? 😉 I can send a follow-up PR if you do, though.

Reply:
Yep I was thinking of doing it myself in a follow up PR, will be much clearer

Reply:
Thanks!